### DISCUSSION TITLE
Vertical Subclaims (bump)

### DISCUSSION THREAD

**Discussion by TheSpotSMP** [GriefPrevention/GriefPrevention#1528] [Ideas]
Hello
I would like to suggest vertical subclaims. Example, being able to claim say a 10x10 area, but also with a Y value of say 10. This way you could stack subclaims, like if you were doing condos/apartments in your little town.

I personally would LOVE to have this feature on my server.
Thanks for reading!

**Comment by RoboMWM** [comment]
You ain't bumpin nothin by making another one
